src/GridFieldAddNewInlineButton.php

```diff
@@ -85,6 +87,28 @@ public function setTitle($title)
         return $this;
     }
 
+    /**
+     * Gets the prepend new inline rows flag.
+     *
+     * @return bool
+     */
+    public function getPrependRows()
+    {
+        return $this->prependRows;
+    }
+
+    /**
+     * Sets the prepend new inline rows flag.
+     *
+     * @param bool $prependRows
+     * @return GridFieldAddNewInlineButton $this
+     */
+    public function setPrependRows($prependRows)
```

Comment:
```suggestion
     */
    public function setPrependRows(bool $prependRows): static
```
New API should have explicit type hints instead of PHPDocs.
If you choose to remove the configuration, ignore this.